src/main/drivers/pwm_output.c

```diff
@@ -394,8 +392,17 @@ void pwmWriteDshotCommand(uint8_t index, uint8_t command)
         }
 
         for (; repeats; repeats--) {
-            motor->requestTelemetry = true;
-            pwmWriteDshotInt(index, command);
+            for (uint8_t i = 0; i < motorCount; i++) {
+                if ((i == index) || (index == ALL_MOTORS)) {
+                    motorDmaOutput_t *const motor = getMotorDmaOutput(i);
+                    motor->requestTelemetry = true;
+                    pwmWriteDshotInt(i, command);
+                } else {
+                    // Needed to avoid DMA errors
+                    pwmWriteDshotInt(i, DSHOT_CMD_MOTOR_STOP);
+                }
```

Comment:
Thought about that too, but `pwmCompleteDshotMotorUpdate` works on timers, not DMA channels, so the channel map to be enabled would have to be constructed on the fly, which is probably not ideal, considering it's timing critical code: 
https://github.com/mikeller/betaflight/blob/91ffa4b88ca6ed0678e840afeaa634d68a339b89/src/main/drivers/pwm_output_dshot.c#L77